src/iotjs_binding.cpp

```diff
@@ -70,34 +70,32 @@ JObject::JObject(const JObject& other) {
 
 
 JObject::JObject(bool v) {
-  _obj_val = JVal::Bool(v);
+  _obj_val = iotjs_jval_bool(v);
```

Comment:
I'd remove these one line wrapper methods and use the jerryscript API here.

`_obj_val = jerry_create_boolean (v);`

So at other places.